doc/source/reference/config.rst

```diff
@@ -65,6 +65,31 @@ If you try to set the default_display to anything other than 'lux' or 'pandas,'
   :width: 700
   :align: center
 
+
+Change the plotting scale 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can set the :code:`plotting_scale` to scale the size of charts displayed in the Lux widget. In the following block, we set plotting scale to 2, therefore the Lux widget will display charts two times larger than the default. 
+
+.. code-block:: python
+
+    lux.config.plotting_scale = 2 
+    df
+
+.. image:: https://github.com/lux-org/lux-resources/blob/master/doc_img/scale-1.png?raw=true
+  :width: 700
+  :align: center
+
+When specifiying intent, the plotting scale applies to both the current visualization and the chart gallery. 
```

Comment:
```
When further operations are performed (e.g., specifying intent), the user-defined plotting scale continues to apply to both the current visualization and recommended charts. 
```

Reply:
This statement and image about specifying intent are not entirely necessary, since there is nothing special about specifying intent from the user's point of view. Instead, it might be cool to showcase the shrinking example.
```
In addition to specifying :code:`plotting_scale` as a positive integer (i.e., >1, enlarging the charts), :code:`plotting_scale` can also take in any positive float between [0,1] to shrink the size of the charts.
[add example]
lux.config.plotting_scale = 0.5 
```